.travis.yml

```diff
@@ -1,30 +0,0 @@
-sudo: false
-
-notifications:
-  email: false
-
-branches:
-  only:
-    - main
-
-os: osx
-osx_image: xcode12
-
-env:
-  - DESTINATION="platform=iOS Simulator,OS=11.4,name=iPhone 7"
-  - DESTINATION="platform=iOS Simulator,OS=12.4,name=iPhone 7"
-  - DESTINATION="platform=iOS Simulator,OS=13.5,name=iPhone 8"
-  - DESTINATION="platform=iOS Simulator,OS=14.0,name=iPhone 8"
-
-script:
-  xcodebuild -scheme "BlueprintUI-Package" test -destination "$DESTINATION"
-
-jobs:
-  include:
-  - name: "Cocoapods BlueprintUI"
-    script: xcodebuild -workspace "SampleApp.xcworkspace" -scheme "BlueprintUI-Unit-Tests" -destination "platform=iOS Simulator,OS=14.0,name=iPhone 8"
-    before_install:
-      - cd SampleApp
-      - bundle install
-      - bundle exec pod repo update
```

Comment:
I wonder if we need to repo update in the new script as well - @watt do you know why this is here?

Reply:
> I wonder if we need to repo update in the new script as well - @watt do you know why this is here?

If we ever add any external pod dependencies this might be needed? Doesn't seem necessary now, though.